target_definition_files/vda_readout_schemes.json

```diff
@@ -1,6 +1,6 @@
 {
     "metadata": {
-        "version": "1.1",
+        "version": "1.1.1",
```

Comment:
Are you tracking version numbers as 
Major.Minor.Patch?

Your previous version number is only two digits. Right now your previous one should be 1.1.0 to be consistent with this numbering scheme. You could just do Major.Minor and make this 1.2

Reply:
I suggest we switch to Major.Minor.Patch numbering where:
- Major = refactor
- Minor = adding new readout schemes
- Patch = changing small number of parameters.